ReactWindows/Playground.Net46/AppReactPage.cs

```diff
@@ -32,6 +31,18 @@ public override bool UseDeveloperSupport
                 return true;
 #else
                 return false;
+#endif
+            }
+        }
+
+        public override bool UseBundleCaching
```

Comment:
This demonstrates overriding the virtual UseBundleCaching flag.   In this example, we opt out of bundle caching if building in a DEBUG configuration.  Otherwise, bundle caching is enabled.